docs/src/pages/component-api/Stepper/Stepper.md

```diff
@@ -0,0 +1,41 @@
+# MobileStepper
```

Comment:
Does this file need to be regenerated? (Several references to MobileStepper...)

Reply:
`yarn docs:api` and we are good :)

Reply:
Fixed in bee14146a